certgen/certProcessor/src/main/java/org/incredible/certProcessor/CertificateFactory.java

```diff
@@ -43,8 +42,9 @@ public CertificateExtension createCertificate(CertModel certModel, String contex
         /**
          *  recipent object
          *  **/
-        compositeIdentityObjectBuilder.setName(certModel.getRecipientName()).setId(uuid).setHashed(true).
-                setType(new String[]{"urn"});
+        compositeIdentityObjectBuilder.setName(certModel.getRecipientName()).setId(certModel.getRecipientPhone())
+                .setHashed(true).
```

Comment:
hash must be true only when phone number is hashed (not visible as plain phone number).